src/modules/mavlink/mavlink_main.cpp

```diff
@@ -609,7 +609,7 @@ Mavlink::mavlink_open_uart(const int baud, const char *uart_name, const bool for
 		}
 	}
 
-	if (_uart_fd < 0) {
+	if (_uart_fd < 0 || _mode == MAVLINK_MODE_IRIDIUM) {
```

Comment:
Add a brief comment perhaps?

Reply:
Sure, something like:
`Return here in the iridium mode since the iridium driver does not support the subsequent function calls`
?

Reply:
Yes, just something simple to make sure someone doesn't misinterpret it as a mistake in the future.

Reply:
done